var/spack/repos/builtin/packages/py-plotly/package.py

```diff
@@ -16,11 +16,18 @@ class PyPlotly(PythonPackage):
 
     license("MIT")
 
+    version("5.20.0", sha256="bf901c805d22032cfa534b2ff7c5aa6b0659e037f19ec1e0cca7f585918b5c89")
+    version("5.19.0", sha256="5ea91a56571292ade3e3bc9bf712eba0b95a1fb0a941375d978cc79432e055f4")
+    version("5.18.0", sha256="360a31e6fbb49d12b007036eb6929521343d6bee2236f8459915821baefa2cbb")
+    version("5.17.0", sha256="290d796bf7bab87aad184fe24b86096234c4c95dcca6ecbca02d02bdf17d3d97")
+    version("5.16.1", sha256="295ac25edeb18c893abb71dcadcea075b78fd6fdf07cee4217a4e1009667925b")
+    version("5.15.0", sha256="822eabe53997d5ebf23c77e1d1fcbf3bb6aa745eb05d532afd4b6f9a2e2ab02f")
```

Comment:
Hi @potter-s: Many thanks!

A question from a spack/maintainer perspective: Would it be enough to add just 5.18 and 5.20? In-between versions can still be built if requested explicitly, and the solver would pick 5.18 as the lasted version for e.g. `python@:3.7`, and `5.20` if `python@3.8:`
```suggestion
    version("5.20.0", sha256="bf901c805d22032cfa534b2ff7c5aa6b0659e037f19ec1e0cca7f585918b5c89")
    version("5.18.0", sha256="360a31e6fbb49d12b007036eb6929521343d6bee2236f8459915821baefa2cbb")
```
Even though the solver is fast now, if these other versions aren't needed, I'd like to skip adding them unless you'd have a reason for adding some or all or them?

Reply:
Anyway, no problem!